test/test_dynamic_shapes.py

```diff
@@ -14,6 +14,18 @@ def test_wrap(self):
     a3 = a2.shape[0] + 3  # tests wrap
     self.assertIsInstance(a3, torch.SymInt)
 
+  def test_sizeAdd(self):
+    size1 = 5
+    size2 = 2
+    t1 = torch.zeros([size1, size2], device=dev)
+    t1[3][0] = 1
+    # t2 has size [<=10, 2]
+    t2 = torch.nonzero(t1)
+    t4 = t2.shape[0] + t2.shape[1]  # Creates a SizeAdd node.
+    dynamic_size = int(t4)  # Exercises SizeAdd::getDynamicValue
```

Comment:
let's move comments above, we don't use comment at the end so it is better to be consistent across whole repo.

Reply:
Will do. 

For `SizeAdd::isSymbolic` and `SizeAdd::Lower`, we can test indirectly by testing `expand_symint`. Is there a way to test `SizeAdd::ToString` from python's perspective?

Reply:
expand a tensor(t5) to t4, and then do 

`print(torch_xla._XLAC._get_xla_tensors_text([t5]))`.

btw don't call it t4, it is not a tensor, but a size. maybe something like `dyn_size` is better